sycl/unittests/kernel-and-program/DeviceInfo.cpp

```diff
@@ -85,3 +94,35 @@ TEST_F(DeviceInfoTest, GetDeviceUUID) {
       << "Expect device UUID to be "
       << "of 16 bytes";
 }
+
+TEST_F(DeviceInfoTest, BuiltInKernelIDs) {
+  if (Plt.is_host()) {
+    return;
+  }
+
+  context Ctx{Plt.get_devices()[0]};
+  TestContext.reset(new TestCtx(Ctx));
+  TestContext->BuiltInKernels = "Kernel0;Kernel1;Kernel2";
+
+  device Dev = Ctx.get_devices()[0];
+
+  auto ids = Dev.get_info<info::device::built_in_kernel_ids>();
+
+  EXPECT_EQ(ids.size(), 3u);
```

Comment:
If this value is less than 3, the following statements will be UB.

```suggestion
  ASSERT_EQ(ids.size(), 3u);
```

Reply:
Changed.